src/components/errorPage/index.tsx

```diff
@@ -66,4 +72,12 @@ const ErrorPage: FC<Props> = ({ statusCode, language, onButtonClick }) => {
   );
 };
 
+const ErrorPage: FC<Props> = (props) => {
+  return (
+    <TranslationProvider language={props.language} scopes={['errorPage']}>
```

Comment:
I would just add it directly to the components instead of creating two components